common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java

```diff
@@ -34,7 +34,7 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
-public class DiagnosisKeyBuilderTest {
+ class DiagnosisKeyBuilderTest {
```

Comment:
Hi @felixbohnacker, could you please remove the leading space here before `class` and also everywhere else below where a leading space was introduced? Thanks!

Reply:
Thank you for the remark, I removed them.